### PR TITLE
added product meta data above the fold

### PR DESCRIPTION
**Pull request by 0xModene**
## **Type of Change**

- [ ] Bug fix
- [x] New feature
- [ ] Documentation
- [ ] Code Refactoring
- [ ] Update of code styling

&nbsp;

## **Does this request include a breaking change?**

- [ ] Yes
- [x] No

&nbsp;

## **Description**

- Added token metadata to the top of the page
    - FLI products get different data than other products
- Widened page content area

&nbsp;

**Summary of Changes:**

**Fixes:** Issue #

&nbsp;

## **Test Data or Screenshots**
Browser:
Screen Shot 2021-06-10 at 9 52 39 PM
Screen Shot 2021-06-10 at 9 52 53 PM

Mobile: 
Screen Shot 2021-06-10 at 9 52 24 PM
Screen Shot 2021-06-10 at 9 51 57 PM

Additionally, the Stats section was moved up underneath the chart, on advice from Dev
Screen Shot 2021-06-10 at 9 53 17 PM

&nbsp;

## **Checklist**

- [x] I have reviewed the [Contribution Guidelines](https://github.com/SetProtocol/index-ui/blob/master/CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [x] I have checked my code and corrected any errors.
- [x] I have commented my code or used an effective naming scheme.
- [x] I have updated my repository to match the master at `SetProtocol/index-ui`.
- [x] I have included test data or screenshots that prove my fix is effective or that my feature works.

&nbsp;

### CODE REVIEW

File: src/components/MarketData/MarketData.tsx [hunk 1]
```diff
@@ -3,40 +3,35 @@ import styled from 'styled-components'
...
-}) => {
+const MarketData: React.FC<MarketDataProps> = ({ tokenData }) => {
   const [chartPrice, setChartPrice] = useState<number>(0)
   const [chartDate, setChartDate] = useState<number>(Date.now())
   const [chartRange, setChartRange] = useState<number>(
     PriceChartRangeOption.MONTHLY_PRICE_RANGE
   ) //default 30 since default chart is 1M
-  const [dateString, setDateString] = useState<String>('')
+  const [dateString, setDateString] = useState<String>(
+    new Date(Date.now()).toDateString()
```
Comment (controtie) [inline src/components/MarketData/MarketData.tsx:25]: nice, this has been driving me crazy

File: src/components/MarketData/MarketData.tsx [hunk 2]
```diff
@@ -3,40 +3,35 @@ import styled from 'styled-components'
 import numeral from 'numeral'
 
 import SimplePriceChart from 'components/SimplePriceChart'
+import { TokenDataProps } from 'components/ProductPage/ProductDataUI'
 
 import { PriceChartRangeOption } from 'constants/priceChartEnums'
 import { InputProps } from 'react-neu'
+import { IndexToken } from 'constants/productTokens'
+import { MetaData } from 'components/ProductPage'
+import IndexComponent from 'components/IndexComponent'
```
Comment (controtie) [inline src/components/MarketData/MarketData.tsx:12]: looks like this is unused

File: src/components/ProductPage/ProductMetaData.tsx [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+      .format('0,0')
+      .toString()
+      .toUpperCase()
+
+  const isFLI =
+    tokenData.token.symbol === Ethereum2xFlexibleLeverageIndex.symbol ||
+    tokenData.token.symbol === Bitcoin2xFlexibleLeverageIndex.symbol
+
+  const realLeverage = tokenData.components
+    ? Number(tokenData.components[0].percentOfSet || 0) / 100
```
Comment (controtie) [inline src/components/ProductPage/ProductMetaData.tsx:34]: I think I'm getting an error here:
`Cannot read property 'percentOfSet' of undefined`. Probably need optional chaining

Comment (0xModene) [reply src/components/ProductPage/ProductMetaData.tsx:34]: hmm... I didn't notice this in testing. I'll try to poke around

**Comment by controtie** [comment]
Can we also add net-asset-value to the FLI pages

**Review by controtie — APPROVED** [review]

**Pull request merged**